packages/apps/composer-app/src/playwright/comments.spec.ts

```diff
@@ -57,12 +57,24 @@ test.describe('Comments tests', () => {
     await Markdown.select(editorTextbox, editorText);
     await Thread.createComment(host.page, plank.locator, messageText);
     const thread = Thread.getThread(host.page, editorText);
-    const message = Thread.getMessage(thread, messageText).getByRole('textbox');
-    await expect(message).toContainText(messageText);
+    const message = Thread.getMessage(thread, messageText);
+    const messageTextbox = message.getByRole('textbox');
+
+    await expect(messageTextbox).toContainText(messageText);
+
+    const editButton = host.page.getByTestId('thread.message.edit');
+    await editButton.click();
 
     const editedText = 'Edited';
-    await message.fill('');
-    await message.fill(editedText);
+
+    for (let i = 0; i <= messageText.length; i++) {
```

Comment:
did `fill('')` stop working?

Reply:
It auto-focuses after clicking the edit button which seemed to break it.